projects/packages/videopress/src/class-stats.php

```diff
@@ -69,7 +69,7 @@ public static function get_today_plays() {
 
 		$data = $response['data'];
 
-		if ( ! $data || ! isset( $data['days'] ) || count( $data['days'] ) === 0 ) {
+		if ( ! $data || ! isset( $data['days'] ) || ! is_countable( $data['days'] ) || count( $data['days'] ) === 0 ) {
```

Comment:
I never know when to replace (or not) `count( $var ) === 0` by `$var === array()`

Reply:
Yeah, I'm not sure about that either. Mostly, I've only been changing those when rector flags it.